CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - Add `voucherCodeBulkDelete` mutation.
   - Adjust voucher usage calculations.
 - Improved GraphQL ID validation messages - #14447 by @patrys
+- Add taxes to undiscounted prices - #14095 by @jakubkuc
```

Comment:
Let's move it and it will be ready to merge.

Reply:
moved to unrealeased 3.19